pkg/kubelet/dockertools/manager.go

```diff
@@ -605,6 +605,7 @@ func (dm *DockerManager) runContainer(
 			MemorySwap: -1,
 			CPUShares:  cpuShares,
 		},
+		ShmSize:     1024 * 1024 * 64,
```

Comment:
We have a new Fedora Docker RPM pending to fix this.

On Thursday, April 21, 2016, Lantao Liu <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/24589#discussion_r60538481>
> :
>
> > @@ -605,6 +605,7 @@ func (dm *DockerManager) runContainer(
> >  			MemorySwap: -1,
> >  			CPUShares:  cpuShares,
> >  		},
> > +		ShmSize:     1024 * 1024 * 64,
>
> I think it would be better to set default value in kube_docker_client
> <https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockertools/kube_docker_client.go#L128>
> .
> Are you OK with that? :)
> If so, I could send a PR right away.
>
> Moreover, I'll file an issue to engine-api to see whether they can set
> ShmSize field to be "omitempty".
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24589/files/97419a0fca88db9ee9af8a131477844b8a1f21cd#r60538481>
>